tests/react/transition.test.tsx

```diff
@@ -51,12 +52,12 @@ describeWithUseTransition('useTransition', () => {
     )
 
     resolve()
-    await findByText('delayed: 0')
+    expect(await findByText('delayed: 0')).toBeTruthy()
```

Comment:
```suggestion
   await findByText('delayed: 0')
```

Reply:
This causes the tests to fail randomly for some reason 🤔. I think we need those matchers in this test

Reply:
Interesting. I thought it behaves the same. Good to know.

Reply:
Does this work?
```suggestion
    await waitFor(() => expect(getByText('delayed: 0')).toBeTruthy())
```

Reply:
Surprisingly yes! 🤔 I thought they worked the same too.